.github/workflows/scripts/build.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3 -u
+
+# -- prioritized --
+import sys
+import os.path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__)))
```

Comment:
agh would be nice if we can avoid manipulating `sys.path` as well :P

Reply:
No, if we remove this hack the script would only work when current directory is at where the script lies. Or we have to cram all the dependencies into a single file which is apparently a worse solution.